README.md

```diff
@@ -318,28 +318,32 @@ the maintainers if you share your changes in a pull request.
 
 Run it with:
 
-    # build image
-    docker build --tag vectorim/riot-web:develop --file Dockerfile.develop .
-
-    # remove old container or do nothing
-    docker rm -f riot || true
-
-    # run a new container with src folder mounted from git repository
-    # located on the host machine. You can mount additional folders
-    # such as res with additional --volume $PWD/res:/src/res
-    docker run --detach --interactive --tty \
-        --publish <port-on-host>:8080 \
-        --volume $PWD/src:/src/src \
-        --name riot \
-        vectorim/riot-web:develop
+```bash
+# build image
+docker build --tag vectorim/riot-web:develop --file Dockerfile.develop .
+
+# remove old container or do nothing
+docker rm -f riot || true
+
+# run a new container with src folder mounted from git repository
+# located on the host machine. You can mount additional folders
+# such as res with additional --volume $PWD/res:/src/res
+docker run --detach --interactive --tty \
+    --publish <port-on-host>:8080 \
+    --volume $PWD/src:/src/src \
```

Comment:
`/riot/src` would still be better, I think.

Reply:
Working on it! :) Hopefully until the end of the week I'll have more time for testing Docker builds, since this will need to change the folder within `Dockerfile` (already present in `master`).

Reply:
Resolved